Headlines/src/ViewModels/CategoriesContainerViewModel.swift

```diff
@@ -39,32 +39,34 @@ class CategoriesContainerViewModel: NSObject {
     func loadImage(with category: Category, at cell: CategoryCollectionViewCell) {
         let key = CategoriesContainerViewModel.imageKeyFromCategory(category)
 
-        if let cachedImage = ImageCacheManager.shared.image(forKey: key) {
-            cell.backgroundImageView.image = cachedImage
-        } else {
-            guard let imageURL = category.imageURL else {
-                return
-            }
-
-            cell.backgroundImageView.alpha = 0
-            cell.backgroundImageView.sd_setImage(with: imageURL) {(image, _, _, _) in
-                guard let img = image else {
+        ImageCacheManager.shared.image(forKey: key) { image in
+            if let cachedImage = image {
+                cell.backgroundImageView.image = cachedImage
+            } else {
+                guard let imageURL = category.imageURL else {
                     return
                 }
-
-                DispatchQueue.global().async {
-                    let tintColor = UIColor(white: 0, alpha: 0.35)
-                    let blurredImage = img.applyBlurWithRadius(3, tintColor: tintColor, saturationDeltaFactor: 1)
-
-                    DispatchQueue.main.async {
-                        cell.backgroundImageView.image = blurredImage
-                        if let blurredImage = blurredImage {
-                            ImageCacheManager.shared.setImage(blurredImage, forKey: key)
+                
+                cell.backgroundImageView.alpha = 0
+                cell.backgroundImageView.kf.setImage(with: imageURL) {(image, _, _, _) in
+                    guard let img = image else {
+                        return
+                    }
+                    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/a5cc79d8b0bc8bc433e47d6a58e00624ade77840/Headlines/src/ViewModels/CategoriesContainerViewModel.swift#L55"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>CategoriesContainerViewModel.swift:55</code></td>
    </tr>
  </tbody>
</table>